test/interactive.py

```diff
@@ -0,0 +1,140 @@
+#! /usr/bin/python2
+
```

Comment:
Can you write some comments about how to use this at the top?  And a link to the pexpect dependency and how you installed it, etc.

Also please follow the style of the other Python scripts

- `#!/usr/bin/env python2` (important for some distros)
-  2 space indents

Not sure exiting with g_failures is a good idea; I think 0 or 1 is more conventional.  If you have 256 failures it will overflow.

Reply:
Would you prefer this as a docstring or just as comments?

Reply:
went with the docstring for now.